### ISSUE TITLE
Internal SimEngine state debug log expresses uuid with individual bytes

### ISSUE THREAD

**Issue by lleshchi** [stratis-storage/stratisd#1784]
As seen below, the debug log is expressing the uuid as a decimal byte-by-byte representation on multiple lines, rather than the standard hex single-line representation. This is inefficient to read or for any sort of uuid comparison, since a conversion to hex would be required. 

```
INFO stratisd: Dump timer expired, dumping state
DEBUG stratisd: Engine state: 
SimEngine {
    pools: {
        (
            "test_pool",
            SimpleRef(
                Uuid(
                    [
                        232,
                        190,
                        118,
                        78,
                        189,
                        175,
                        73,
                        180,
                        179,
                        149,
                        173,
                        18,
                        144,
                        210,
                        29,
                        255,
                    ],
                ),
            ),
        ): SimPool {
            block_devs: {
                Uuid(
                    [
                        185,
                        195,
                        66,
                        136,
                        180,
                        230,
                        65,
                        56,
                        139,
                        68,
                        190,
                        164,
                        27,
                        252,
                        190,
                        236,
                    ],
                ): SimDev {
                    devnode: "junk",
                    rdm: RefCell {
                        value: {Randomizer 0,
                    },
                    user_info: None,
                    hardware_info: None,
                    initialization_time: 1579119473,
                    dbus_path: MaybeDbusPath(
                        Some(
                            Path(
                                "/org/storage/stratis2/6",
                            ),
                        ),
                    ),
                },
            },
```

**Comment by mulkieran** [comment]
The reason this happens is that the output of the log is simply a pretty print of the Debug output of the whole engine. The Debug format for UUIDs is defined to show each byte individually. Hence the verbose output.

**Comment by mulkieran** [comment]
Related: https://github.com/stratis-storage/project/issues/93.

**Comment by mulkieran** [comment]
This was resolved by #1907, #1979 will not suffer from the same defect.
